### DISCUSSION TITLE
Labels for physical buttons

### DISCUSSION THREAD

**Discussion by filmgarage** [Blackymas/NSPanel_HA_Blueprint#2174] [Q&A] [answered]
On the buttons page the labels for the physical buttons do not show up. The status bar does show however... Is it possible to show the text label too? Is there a setting for this in the blueprint or am I missing something?

**Comment by madnat** [comment]
This works fine for me - set the desired text for the physical button and entity to control in thie section named: ✅ HARDWARE BUTTONS ✅ in the blueprint

**Comment by edwardtfn** [comment, marked as answer]
The labels are shown only on the home page.
Most of the pages we don't have space available for those labels, unfortunately.

**Reply by filmgarage** [reply]
Thanks, I thought it might fit still. But I guess it gets a little too crowded at the bottom of the page alongside the buttons... :-)

**Reply by edwardtfn** [reply]
In fact, the button pages probably still have some space for that if we move the buttons a bit higher. I will take a look at that.
Ar you using US or EU version?

**Reply by filmgarage** [reply]
It's the EU version I am using and there is still some space as far as I can judge. If the button labels can still be enaled/disabled on this page it would be up the user...
